### PR TITLE
Add korean translation

### PR DESCRIPTION
**Pull request by Astro36**

### CODE REVIEW

**Comment by k0shk0sh** [comment]
please fix below:

```
/home/travis/build/k0shk0sh/FastHub/app/src/main/res/values-ko/strings.xml:419:42: Error: Element type "string" must be followed by either attribute specifications, ">" or "/>".
```

File: app/src/main/res/values-ko/strings.xml [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+    <string name="warning">경고</string>
+    <string name="owner">소유자</string>
+    <string name="original_poster">원본 포스터</string>
+    <string name="cancel_reviews">검토 취소</string>
+    <string name="colored_navbar_summary">테마에서 색상 네비게이션 바를 비활성화합니다</string>
+    <string name="colored_navbar_title">무색 네비게이션</string>
+    <string name="sound_chooser_summary">알림 소리를 선택합니다</string>
+    <string name="sound_chooser_title">알림 소리 선택</string>
+    <string name="disable_auto_gif_summary">GIF 자동 재생 사용을 비활성화합니다</string>
+    <string name="disable_auto_gif_title"GIF 자동 재생 사용 비활성화</string>
```
Comment (k0shk0sh) [inline app/src/main/res/values-ko/strings.xml]: there is `<` is missing

Comment (yakov116) [reply app/src/main/res/values-ko/strings.xml]: @k0shk0sh its a `>`

Comment (k0shk0sh) [reply app/src/main/res/values-ko/strings.xml]: @yakov116 thanks :D

**Comment by Astro36** [comment]
Sorry! Now it is fixed.

**Pull request merged**